packages/synapse-react-client/src/components/SynapseTable/SynapseTableRenderers.tsx

```diff
@@ -203,11 +203,7 @@ export function TableDataColumnHeader(
       <div className="SRC-centerContent" style={{ height: '22px' }}>
         {isFacetSelection && !isLockedColumn && columnModel && (
           <span>
-            <EnumFacetFilter
-              containerAs="Dropdown"
-              facetValues={facet.facetValues}
-              columnModel={columnModel}
-            />
+            <EnumFacetFilter containerAs="Dropdown" facet={facet} />
```

Comment:
Simplified `EnumFacetFilter` prop signature; just pass the facet, it will figure out the columnModel internally (so we don't have to do the same thing all over the place)